src/java/org/apache/cassandra/db/ColumnFamilyStore.java

```diff
@@ -1456,6 +1457,12 @@ public void apply(PartitionUpdate update, CassandraWriteContext context, boolean
         }
         catch (RuntimeException e)
         {
+            if (e instanceof InvalidRequestException)
+            {
+                throw new InvalidRequestException(e.getMessage()
```

Comment:
The message construction is identical for `InvalidRequestException` and `RuntimeException`. We could use:
```java
catch (RuntimeException e)
{
    String msg = e.getMessage() + " for ks: " + getKeyspaceName() + ", table: " + name;

    if (e instanceof InvalidRequestException)
        throw new InvalidRequestException(msg, e);
    
    throw new RuntimeException(msg, e);
}
```